suite_cam4.xml

```diff
@@ -0,0 +1,204 @@
+      <scheme>diff_flux_diag</scheme>                 <!-- Calculate diffusion variable diagnostics, if do_pbl_diags is False (is that correct?) -->
+      <scheme>diff_flux_tend_dry</scheme>             <!-- Calculate vertical diffusion tendencies for "dry" quantities, and send them back to host model -->
+      <scheme>set_wet_to_dry</scheme>                 <!-- Convert mixing ratios from air + water vapor to dry air -->
+      <scheme>pbl_diag_calc</scheme>                  <!-- Calculate PBL diagnositc variables, if do_pbl_diags is False -->
+      <scheme>diff_mass_check</scheme>                <!-- Check mass conservation after vertical diffusion, and send error messages if check fails -->
+      <scheme>pbl_output</scheme>                     <!-- Write PBL diagnostic variables to history files -->
+      <!-- **************************** -->
+      <scheme>rayleigh_friction_tend</scheme>         <!-- Calculate Rayleigh friction tendency -->
+      <scheme>check_energy_chng</scheme>              <!-- Global integral checker required for certain diagnostic outputs -->
+      <scheme>check_tracers_chng</scheme>             <!-- Global tracer mass checker that kills model if error is large enough. -->
+      <scheme>charge_balance</scheme>                 <!-- Enforce charge nuetrality, if chemistry simulates electrons -->
+      <!-- Gravit wave tendencies -->
+      <!-- **************************** -->
+      <!-- This interface depends heavily
+           on namelist inputs, with entire
+           blocks of code being used (or not)
+           depending on said namelist choices.
+           However, the general structure of these
+           code blocks is more or less the same,
+           so a "generic" scheme set is included
+           here, which will need to be expanded
+           out once "if-statement" like dependencies
+           can be added to suite definition files
+           like this one -->
+      <scheme>set_dry_to_wet</scheme>                  <!-- Convert mixing ratios from dry air to air + water vapor -->
+      <scheme>gw_prof</scheme>                         <!-- Calculate layer interface densities and Brunt-Viasala frequencies (combine with other schemes?) -->
+      <!-- Generic gravity wave scheme order  -->
+      <!-- +++++++++++++++++++++++++ -->
+      <scheme>effgw_calc</scheme>                      <!-- Calculate the efficiency of gravity wave momentum transfer -->
+      <scheme>gw_src_calc</scheme>                     <!-- Determine gravity wave sources -->
+      <scheme>gw_drag_prof</scheme>                    <!-- Solve for the drag profile with the gravity wave spectrum -->
+      <scheme>calc_taucd</scheme>                      <!-- Project stress into directional components -->
+      <scheme>egwdffi_tot_calc</scheme>                <!-- add diffusion coefficients (manageable by registry/framework?) -->
```

Comment:
Not sure.

Reply:
No worries.  I've gone ahead and removed the question for now.